src/games/strategy/triplea/ui/MapRouteDrawer.java

```diff
@@ -336,17 +338,17 @@ private void createMovementLeftImage(BufferedImage image, String curMovement, St
    * @param to The {@linkplain Point2D} where the arrow is placed
    * @return A transformed Arrow-Shape
    */
-  private Shape createArrowTipShape(Point2D from, Point2D to) {
+  private static Shape createArrowTipShape(Point2D from, Point2D to) {
     final Polygon arrowPolygon = new Polygon();
+    // 1- (-3) = arrowLength
```

Comment:
Extract the one to a constant, then you can calculate the -3 value.
Idea is to capture any relationships between lines of code that are known to exist, so everything that needs to change, changes when you change the one thing.